### ISSUE TITLE
Handle multiple comma separated actions.

### ISSUE THREAD

**Issue by rsarky** [pvdlg/issue-parser#9]
## Parser configuration

`parse('github')`

## Text to parse

`Fixes #1,#2`

## Current Behavior

```json  
{ actions:
   [ { raw: 'Fixes #1',
       action: 'Fixes',
       slug: undefined,
       prefix: '#',
       issue: '1' } ],
  refs: [ { raw: '#2', slug: undefined, prefix: '#', issue: '2' } ],
  duplicates: [],
  mentions: [] }
```

## Expected behavior

Should return 2 actions instead of 1 action and 1 ref.

**Comment by pvdlg** [comment]
I don't think so. In a GitHub PR if you add `Fixes #1,#2`, when merging the PR only `#1` will be closed.
So GitHub in such case GitHub applied the `Fixes` keyword only to the first one.

See https://github.com/pvdlg/test-sr/pull/10. If you hover your cursor above the `Fixes` GitHub will show a popup saying only `#10` will be closed.

**Comment by rsarky** [comment]
Hmm makes sense.
I guess this could be something GitHub could look into?
IMO `Fixes #1,#2` is more concise and apt as opposed to `Fixes #1 Fixes #2` 
What do you think?

**Comment by pvdlg** [comment]
The syntax to close multiple issues is explicitly mentioned in the GitHub doc: https://help.github.com/articles/closing-issues-using-keywords/#closing-multiple-issues.
So I guess this is intentional and thought through. So I don't think GitHub would change it.

Regarding `issue-parser` the objective is to exactly mimic GitHub so one can find out from the description of a PR what issue was or will be closed on merge. So independently of opinions and preferences about the syntax, the reference is how GitHub (or GitLab, Bitbucket) parse issues/PRs.
As the behavior is expected I'll close the issue.
